### PR TITLE
Implement KITCO requested changes

### PR DESCRIPTION
**Pull request by chauhannishith**
account page

- [x] adjust spacing for password
- [x] adjust spacing for 2FA
- [x] adjust content for password
- [x] adjust content for 2FA

home page
- [x] adjust spacing

login page

- [x] add stably logo
- [x] adjust text size
- [x] add background

add eth page
- [x] adjust spacing

add bank page
- [x] adjust spacing

- [x] fix bug where redirect happening after applied for verification

### CODE REVIEW

File: ts/kgld/Web/src/components/account/account.tsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import React from 'react'
-import context from '../../util/localize'
+// import context from '../../util/localize'
```
Comment (Git-on-my-level) [inline ts/kgld/Web/src/components/account/account.tsx:2]: do we need to keep this comment here?

File: ts/kgld/Web/src/container/EthereumAccount.tsx [hunk 2]
```diff
@@ -71,11 +71,13 @@ const EthereumAccountContainer: React.FC<ContainerRouterProps> = (props: any): a
       try {
         setLoading(true)
         //redirect if not applied for verification
-        const reply = await api.firstdigital.appliedForVerification()
-        if (!reply.applied_for_verification) {
-          props.history.push('/kyc')
+        const applied = await api.firstdigital.appliedForVerification()
+        console.log(applied, 'applied')
```
Comment (Git-on-my-level) [inline ts/kgld/Web/src/container/EthereumAccount.tsx:75]: log

File: ts/kgld/Web/src/container/EthereumAccount.tsx [hunk 3]
```diff
@@ -71,11 +71,13 @@ const EthereumAccountContainer: React.FC<ContainerRouterProps> = (props: any): a
...
+        if (!applied) {
```
Comment (Git-on-my-level) [inline ts/kgld/Web/src/container/EthereumAccount.tsx:76]: what does this if do?

Comment (chauhannishith) [reply ts/kgld/Web/src/container/EthereumAccount.tsx:76]: missed to uncomment it

**Pull request merged**